azure-batch/HISTORY.rst

```diff
@@ -3,6 +3,16 @@
 Release History
 ===============
 
+2.0.0 (2017-02-17)
+++++++++++++++++++
+
+- AAD token authentication now supported.
+- Support for running tasks under a predefined or automatic user account.
```

Comment:
Also, do we want to clarify that the run as user changes apply to all "task-y" things (task, job manager task, start task, job prep, job release), but the scoped auth only applies to task and job manager task?